app/src/main/java/com/instructure/template/projectCodeHere/ProfilePage.java

```diff
@@ -0,0 +1,184 @@
+package com.instructure.template.projectCodeHere;
+
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.Icon;
+import android.net.Uri;
+import android.os.AsyncTask;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.navigation.fragment.NavHostFragment;
+import com.instructure.template.R;
+import com.instructure.template.loginTemplate.api.ApiPrefs;
+import com.instructure.template.loginTemplate.api.apiHelpers.CanvasRestAdapter;
+import com.instructure.template.loginTemplate.login.LoginActivity;
+import org.jetbrains.annotations.NotNull;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class ProfilePage extends Fragment {
+
+    private TextView textView;
+    private Button logoutButton;
+    private ImageView imageView;
+    private TextView textView2;
+    private TextView textView3;
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.profile_page, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        logoutButton = view.findViewById(R.id.logoutButton);
+        textView = view.findViewById(R.id.textView);
```

Comment:
Make these variable names a little more descriptive. Right click where you're defining it and `Refactor -> Rename...`